src/navigation/services/buy-crypto/screens/BuyCryptoOffers.tsx

```diff
@@ -449,6 +463,15 @@ const BuyCryptoOffers: React.FC = () => {
   const getSimplexQuote = (): void => {
     logger.debug('Simplex getting quote');
 
+    if (buyCryptoConfig?.simplex?.disabled) {
+      let err = buyCryptoConfig?.simplex?.disabledMessage
+        ? buyCryptoConfig?.simplex?.disabledMessage
+        : t("Can't get rates at this moment. Please try again later");
+      const reason = 'simplexGetQuote Error. Exchange disabled from config.';
```

Comment:
translation ?

Reply:
It is not necessary to translate this string as it is only used for logs and analytics. It is never shown to the user